bciers/apps/reporting/src/app/components/attachments/AttachmentsForm.tsx

```diff
@@ -72,6 +94,28 @@ const AttachmentsForm: React.FC<Props> = ({
   };
 
   const handleSubmit = async (canContinue: boolean) => {
+    if (isSupplementaryReport) {
+      const formDataSupplementary = new FormData();
+      formDataSupplementary.append(
+        "confirm_supplementary_required_attachments_uploaded",
+        confirmRequiredAttachmentsUploaded ? "true" : "false",
+      );
+      formDataSupplementary.append(
+        "confirm_supplementary_existing_attachments_relevant",
+        confirmExistingAttachmentsRelevant ? "true" : "false",
+      );
+
+      const confirmResponse = await postAttachmentConfirmation(
+        version_id,
+        formDataSupplementary,
+      );
+      if (confirmResponse.error) {
+        setErrors([confirmResponse.error]);
+        setIsSaving(false);
+        return;
+      }
+    }
+
```

Comment:
why not add these to the `formData` object, and submit everything at once? This would reduce the risk of getting an inconsistent database in case of error, we can have the save function for these boolean values part of the same transaction as saving the files